frontend/src/pages/ArtifactList.tsx

```diff
@@ -23,7 +23,7 @@ import {
   ArtifactType,
   getArtifactCreationTime,
   getArtifactTypes,
-  getResourceProperty,
+  getResourcePropertyViaFallBack,
```

Comment:
Wouldn't it make more sense to use: `getArtifactName()`

Reply:
It would for the `Name` field, but not for the `Pipeline/Workspace Name` field, which is what's broken in the screenshot.

I agree with using `getArtifactName()`, but it's not exported so it would block this PR on another kubeflow/frontend PR. Don't want to block KFP on that optimization with the release so close.

Reply:
I see, okay no worries!

Reply:
SG, thanks!